kitchen.resources.yml

```diff
@@ -172,6 +172,7 @@ suites:
     verifier:
       controls:
         - dns_domain_configured
+        - dns_domain_setup
```

Comment:
here, let's call the :configure explicitly.

Reply:
not needed in this particular point, but yes, I'll call explicitly the configure where needed after changing the default action